package.json

```diff
@@ -48,23 +48,22 @@
   "dependencies": {
     "@emotion/react": "^11.4.0",
     "@emotion/styled": "^11.3.0",
-    "@govtechsg/address-identity-resolver": "^1.4.1",
```

Comment:
Are there any references to this package? I searched but couldnt really find.

Reply:
if no hav can remove. i think might be leftover from long time back when it wasn't in ui-components